apps/src/redux/instructions.js

```diff
@@ -0,0 +1,76 @@
+var _ = require('../lodash');
+
+var SET_INSTRUCTIONS_IN_TOP_PANE = 'instructions/SET_INSTRUCTIONS_IN_TOP_PANE';
+var TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
+var SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
+var SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
+
+var instructionsInitialState = {
+  collapsed: false,
+  // represents the uncollapsed height
+  height: 300,
+  maxHeight: 0,
+  inTopPane: false
+};
+
+module.exports.default = function reducer(state, action) {
```

Comment:
I see what you mean about following the duck-module pattern here.  I wasn't sure how to map a default export to ES5.  I'll update my PR to match.